outputs.tf

```diff
@@ -1 +1,4 @@
 # Outputs placeholder
+output "scp_policy_json" {
+  value = data.aws_iam_policy_document.combined_policy_block.json
+}
```

Comment:
Why are you adding this as an output?

Reply:
Good eye. It was helping me debug / see if the malformed policy error was literally a json problem or not. I'll take it out.